{"name":"Solicitudes concurrentes para clientes","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"El cliente no disponible CUSTOMER2 no fue encontrado en la respuesta.","trace":"java.lang.AssertionError: El cliente no disponible CUSTOMER2 no fue encontrado en la respuesta.\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.validarClienteNoDisponible(ValidacionDeUsuarios.java:335)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.elSistemaRespondeQueNoEstanDisponibles(ValidacionDeUsuarios.java:257)\r\n\tat ✽.\"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles(file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature:11)\r\n"},"stage":"finished","description":"  Validar si un cliente está disponible para el servicio de detección de phishing.","steps":[{"name":"Given  que el servicio RabbitMQ está corriendo","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735840324134,"stop":1735840326646},{"name":"And  la base de datos PostgreSQL está activa","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735840326650,"stop":1735840326878},{"name":"When  se realizan solicitudes simultáneas para \"CUSTOMER1\", \"CUSTOMER2\", \"CUSTOMER3\", \"CUSTOMER4\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735840326879,"stop":1735840328085},{"name":"Then  el sistema responde que \"CUSTOMER1\" está disponible","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735840328085,"stop":1735840328121},{"name":"And  \"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"El cliente no disponible CUSTOMER2 no fue encontrado en la respuesta.","trace":"java.lang.AssertionError: El cliente no disponible CUSTOMER2 no fue encontrado en la respuesta.\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.validarClienteNoDisponible(ValidacionDeUsuarios.java:335)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.elSistemaRespondeQueNoEstanDisponibles(ValidacionDeUsuarios.java:257)\r\n\tat ✽.\"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles(file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature:11)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735840328121,"stop":1735840328137}],"attachments":[],"parameters":[],"start":1735840324116,"stop":1735840328145,"uuid":"af390c77-a0d1-4055-a38f-fef7852d7325","historyId":"3ec2e9af468c60667826fb1df283ab","fullName":"Validación de Disponibilidad de Cliente: Solicitudes concurrentes para clientes","labels":[{"name":"host","value":"VandV"},{"name":"thread","value":"22632@VandV.Test worker(1)"},{"name":"feature","value":"Validación de Disponibilidad de Cliente"},{"name":"story","value":"Solicitudes concurrentes para clientes"},{"name":"suite","value":"Validación de Disponibilidad de Cliente"},{"name":"testClass","value":"Solicitudes concurrentes para clientes"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature"},{"name":"package","value":"src.test.resources.features.ValidacionDeUsuarios_feature.Validación de Disponibilidad de Cliente"}],"links":[]}